server/server.js

```diff
@@ -1,15 +1,60 @@
 const express = require('express');
 const path = require('path');
 const bodyParser = require('body-parser');
+const helper = require('../db/dbHelpers.js');
 
 const PORT = 3003;
 const app = express();
 
-
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(express.static(path.join(__dirname, '../public')));
 
-// app.get('/', (req, res) => res.send('Hello World!'));
+app.use((req, res, next) => {
+  console.log(req.body, req.path);
+  next();
+});
+
+app.get('/:id', (req, res) => {
```

Comment:
Again - I'm not totally sure about this, but I think you might want to use something like /businesses/:id/info

Reply:
You right.